### DISCUSSION TITLE
ARM64 Support?

### DISCUSSION THREAD

**Discussion by dashaun** [cfengine/core#4731] [Q&A] [answered]
Is ARM64 support on the roadmap?

I could not find any issues, discussions or wiki pages around ARM.

The supported platforms page doesn't mention it either.

**Comment by olehermanse** [comment]
@dashaun We have customers using CFEngine on ARM64 and I think there are open source users using ARM64 as well (building it themself). We have no immediate plans of adding it to the download tables and the website, it's not a part of our CI system currently, but this could change. Are you using ARM64 on your infrastructure today, or planning to start? Please get in touch using the contact form, and we can support it for you as part of a customer agreement:

https://cfengine.com/contact/

**Comment by craigcomstock** [comment]
@dashaun if you are running something like raspbian it is certainly possible to just install community via source. It takes a little while but not too bad. I have kept the dependencies up to date here: https://github.com/cfengine/core/blob/master/INSTALL

```
* Debian (Raspbian 10 2021-07-02)

# apt-get install -y build-essential git libtool autoconf automake bison flex libssl-dev libpcre3-dev libbison-dev libacl1 libacl1-dev lmdb-utils liblmdb-dev libpam0g-dev libtool libyaml-dev
```

and then 

```
From git checkout:

$ ./autogen.sh [configure options]
$ make [-jN]
```

**Comment by cody-valle** [comment, marked as answer]
@dashaun A quick update on this. We're now building for ARM64 and will be demoing this later today on 'The Agent Is In' webinar. You can use the link below to register. 

https://us02web.zoom.us/webinar/register/WN_JPz9ADvBRS2Ih8SaT0z2Uw

**Reply by dashaun** [reply]
I missed the episode, but I plan to take this for a spin very soon!

**Reply by nickanderson** [reply]
Speaking of episodes .... 

The next episode of The Agent is In is tomorrow, same agent time, same agent place ;)
